module uim.databases.exceptions;

public {
    import uim.databases.exceptions.exception:
}